### PR TITLE
Add electron options to bug-report.yml

### PR DESCRIPTION
**Pull request by Shazib**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ body:
         - Docker
         - Fly.io
         - NAS
+        - Electron App
```
Comment (MatissJanis) [inline .github/ISSUE_TEMPLATE/bug-report.yml:51]:
```suggestion
        - Desktop App (Electron)
```

A lot of folks wont know what "electron" is. All they know is they have a program on their computer.

**Review by MatissJanis — APPROVED** [review]
Great! Thanks!

**Review by j-f1 — CHANGES_REQUESTED** [review]
Can you also add Electron to the browser options? Want to be able to capture issues both with running Electron with a separate server and without.

**Review by j-f1 — APPROVED** [review]
Thanks!

**Pull request merged**